### DISCUSSION TITLE
Request for help adding an additional user as an editor

### DISCUSSION THREAD

**Discussion by amponce** [avitorio/outstatic#83] [Q&A] [answered]
Hi, I recently added a new user to my GitHub repository that contains my out-static blog. The user has successfully authenticated with their GitHub account, but they are unable to view the posts and pages that I have created. When they try to create a new collection, they receive an error message after hitting submit.

I would like to request help in resolving this issue so that the new user can access and edit the existing content in the repository. Thank you in advance

**Comment by avitorio** [comment, marked as answer]
Hi Aaron,
Thanks for reaching out. Are they logging in from a local install or on a
deployed site (vercel, netlify)?

**Reply by amponce** [reply]
Hi Andre, I'm working from a deployed site via Vercel(using a slightly modified version of the outstatic example site). I'm the developer and the user I'm trying to give access to is a non technical writer, so I don't think running anything local is an option for him. 

The domain is:
news.rfpenergysolutions.com

Thanks for the prompt reply!

**Reply by avitorio** [reply]
Is the `OST_REPO_OWNER` env variable set up?

**Reply by amponce** [reply]
Oh no, it wasn't set up previously. I just added it to Vercel using my GitHub username. Do you think that could be the issue? Our writer will be testing it shortly.

**Reply by amponce** [reply]
Outstatic is awesome BTW, nice work!

**Reply by avitorio** [reply]
I think this might solve it. If the `OST_REPO_OWNER` variable isn't setup Outstatic uses the username of the logged in user to look for the repository.

Let me know if it works.

**Reply by amponce** [reply]
Unfortunately, defining the `OST_REPO_OWNER` didn't solve the issue, which was surprising as it seemed like a logical solution. I've taken a screenshot of my current environment variables in Vercel and added the output path, just in case that could help resolve the problem. Do you see anything missing that could be causing the issue?

**Reply by avitorio** [reply]
That's weird. I just managed to login to a test server with a second account and I can perform all the regular user actions.
Please, check [this link ](https://outstatic.com/docs/environment-variables)to see if any of the missing variables could help.

I suggest you try to create a second Github account and see if you can replicate the issue. Check the `Network` tab in your browser to see if the Graphql calls give you any hints. Please, let me know what you find. Thanks!

**Reply by amponce** [reply]
I created a test user as suggested and based on the GraphQL error, the code is functioning as expected, but it's trying to find the repository in the wrong location. The error message is as follows:

```
{
    "data": {
        "repository": null
    },
    "errors": [
        {
            "type": "NOT_FOUND",
            "path": [
                "repository"
            ],
            "locations": [
                {
                    "line": 2,
                    "column": 3
                }
            ],
            "message": "Could not resolve to a Repository with the name '0xAmp/market-intel'."
        }
    ]
}
```
Is it possible that the values of the .env variables are not defined correctly? The repository should be: `amponce/market-intel` 

<img width="950" alt="Screenshot 2023-02-08 at 11 14 16 AM" src="https://user-images.githubusercontent.com/3916436/217629482-2b548580-e886-4553-a6de-2569ed487e12.png">


Thank you for working with me on this, I appreciate your help!

**Reply by avitorio** [reply]
Yeah, in fact, in the `OST_REPO_SLUG` you only need to add `market-intel`. Let me know if this works.

**Reply by amponce** [reply]
After updating the slug and still no luck, I had the realization that might be a caching issue with the previous build .env values. Looks like the redeploy of the site did the trick! Everything is working! Thank you for your help! I honestly think your first suggestion to update the `OST_REPO_OWNER ` would have resolved my issue if I had only thought to redeploy immediately after making the change 🤦

**Reply by avitorio** [reply]
No problem! :)

**Reply by avitorio** [reply]
In the example above:

OST_REPO_SLUG=market-intel
OST_REPO_OWNER=amponce
